### DISCUSSION TITLE
Response can't be cached

### DISCUSSION THREAD

**Discussion by paulhaem** [pimcore/pimcore#16020] [Q&A / Support]
Hello, when inspecting the reponse headers I always see that `X-Pimcore-Output-Cache-Disable-Reason: Response can't be cached` is set, despite Full Page Cache being configured correctly. I dont know about the cache configuration used for the Pimcore demo but it also outputs the same response headers on https://demo.pimcore.com/.

Any ideas what could lead to such a response?

**Comment by dpfaffenbauer** [comment]
as soon as you start a session, the full page cache is disabled. do you have a session?

**Reply by paulhaem** [reply]
Thanks for the quick reply. No, I am not authenticated. Afaik the response would change to `Session in use` in the cache disable reason in case of a existing session: https://github.com/pimcore/pimcore/blob/11.x/bundles/CoreBundle/src/EventListener/Frontend/FullPageCacheListener.php#L310

A few lines above the `Response can't be cached` check happens: https://github.com/pimcore/pimcore/blob/11.x/bundles/CoreBundle/src/EventListener/Frontend/FullPageCacheListener.php#L306

**Reply by paulhaem** [reply]
For some reason the response always includes the `no-cache` header. I'll have a closer look to find out what causes the `no-cache` header to be set.

**Comment by paulhaem** [comment]
Submitted an issue for this: https://github.com/pimcore/pimcore/issues/16057
